Sources/Playbook/Components/Tab Bar/PBTabBar.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  PBTabBar.swift
+//  
+//
+//  Created by Rachel Radford on 12/15/23.
+//
+
+import SwiftUI
+
+public struct PBTabBar: View {
+  @State var isTabSelected: Bool
+  let variant: Variant
+  
+  public init(
+    isTabSelected: Bool = false,
+    variant: Variant = .home
+  ) {
+    self.isTabSelected = isTabSelected
+    self.variant = variant
+  }
+  
+  public var body: some View {
+    tabButtonView
+  }
+}
+
+public extension PBTabBar {
+  enum Variant {
+    case home, calendar, notifications, search, more
```

Comment:
Hey love. I didn't understand what are this variants? The Icons on the handoff are only examples. This tab bar component should be able to receive what ever icon the user choose.

Reply:
ok, I will change this